src/nvim/testdir/test_regexp_latin.vim

```diff
@@ -0,0 +1,18 @@
+" Tests for regexp in latin1 encoding
+set encoding=latin1
+scriptencoding latin1
+
+func Test_nested_backrefs()
```

Comment:
Delete lines 2-3, then add this line here:

    throw 'skipped: Nvim does not support encoding=latin1'

(We want to keep the test file so that in the future someone doesn't have to backtrack to figure out why it wasn't added.)